.github/workflows/azure_e2e_release_promotion_tests.yml

```diff
@@ -0,0 +1,190 @@
+name: Azure E2E Release Promotion Tests
+'on':
+  push:
+    branches:
+      - aj-1436-e2e-staging
+  workflow_dispatch:
+    inputs:
+      testEnv:
+        description: >-
+          Environment in which tests should be run. Currently only runs on
+          staging
+env:
+  BEE_NAME: >-
+    ${{ github.event.repository.name }}-${{ github.run_id }}-${{
+    github.run_attempt }}-dev
+  TOKEN: '${{ secrets.BROADBOT_TOKEN }}'
+  BEE_CREATE_RUN_NAME: >-
+    bee-create-${{ github.event.repository.name }}-${{ github.run_id }}-${{
+    github.run_attempt }}
+  ATTACH_BP_TO_LZ_RUN_NAME: >-
+    attach-billing-project-to-landing-zone-${{ github.event.repository.name
+    }}-${{ github.run_id }}-${{ github.run_attempt }}
+jobs:
+  init-github-context:
+    runs-on: ubuntu-latest
+    outputs:
+      branch: '${{ steps.extract-inputs.outputs.branch }}'
+      delete-bee: '${{ steps.extract-inputs.outputs.delete-bee }}'
+    steps:
+      - name: Get inputs or use defaults
+        id: extract-inputs
+        run: >
+          echo "branch=${{ inputs.branch || 'develop' }}" >> "$GITHUB_OUTPUT"
+
+          echo "delete-bee=${{ inputs.delete-bee || false }}" >>
+          "$GITHUB_OUTPUT"
+  params-gen:
+    runs-on: ubuntu-latest
+    outputs:
+      project-name: '${{ steps.gen.outputs.project_name }}'
+      bee-name: '${{ env.BEE_NAME }}'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate a random billing project name
+        id: gen
+        run: |
+          project_name=$(echo "tmp-billing-project-$(uuidgen)" | cut -c -30)
+          echo "project_name=${project_name}" >> $GITHUB_OUTPUT
+  create-bee-workflow:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Echo Leo version
+        run: >
+          echo '${{ needs.leo-build-tag-publish-job.outputs.custom-version-json
+          }}'
+      - name: dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v4.0.0
+        with:
+          run-name: '${{ env.BEE_CREATE_RUN_NAME }}'
+          workflow: bee-create
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: '${{ env.TOKEN }}'
+          inputs: >-
+            { "run-name": "${{ env.BEE_CREATE_RUN_NAME }}", "bee-name": "${{
+            env.BEE_NAME }}", "bee-template-name": "rawls-e2e-azure-tests", 
```

Comment:
I know this has come up in another PR as a question I didnt know the answer too and I think you were OOO then - but whats the reason behind us using the "rawls" template name?

Reply:
Mostly copy-paste!  It's what all the e2e tests being called are using now so I figured I'd stick with that for this PR.  If any of the underlying tests change the template then we'll have to figure something out

Reply:
I think the rawls template was added so that Janitor can monitor resources used in these run and be able to clean them up (in the case workspace deletion fails)